main.cpp

```diff
@@ -26,13 +30,100 @@ int main()
      *  And then mark that the update succeeded
      */
     //run_self_test();
-    boot_set_confirmed();
+    mbed_trace_init();
+
+    if (boot_set_confirmed() == 0) {
+        printf("Boot confirmed\n");
```

Comment:
Why not use mbed trace in this file since it is enabled?

Reply:
Updated with mbed trace